keras_cv/layers/__init__.py

```diff
@@ -17,6 +17,10 @@
 from tensorflow.keras.layers import RandomWidth
 
 from keras_cv.layers.augmenter import Augmenter
+from keras_cv.layers.detectron2_layers import AddPositionalEmbedding
```

Comment:
Since this would be exported as part of the public API - we have `PatchingAndEmbedding` which does patching with a Conv2D and then adds embeddings in this same form. Do we want to update that layer to use `AddPositionalEmbedding` as well for conformity?

Reply:
Wouldn't adding a new layer in a pre-existing class invalidate the weights set for the ViT model? Also, since `PatchingAndEmbedding` is still a TensorFlow Keras layer, I think, for the time being, it'd be easier to keep the two separate.

Although, I'd add a comment about this as a TODO so we don't forget to do it in the future. What do you think?

Reply:
Done.